.github/workflows/run_frontend_build.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       - name: Install Dependencies
         run: |
           cd frontend/  
+          npm install
           npm ci --omit=dev
```

Comment:
But the issue about `npm ci` is that it only looks for `package-lock.json`, and I don't know why but for some reasons the GitHub Actions is not recognizing my `package-lock.json`... I pretty sure the `package-lock.json` is up-to-date because I purposely deleted the file and run `npm install` locally to generate a new `package-lock.json` file. So to pass the frontend build check, I have to forcefully do `npm install` instead of `npm ci`. But I know that `npm ci` is a good practice in doing CI/CD, so my suggestion is that we leave this for now and continue observe in the next few PRs? Thoughts?

Reply:
I see